pkg/gits/helpers.go

```diff
@@ -341,14 +341,14 @@ func PushRepoAndCreatePullRequest(dir string, upstreamRepo *GitRepository, forkR
 		}
 		gha.Body = fmt.Sprintf("%s\n<hr />\n\n%s", prDetails.Message, existingPr.Body)
 		var err error
-		pr, err = provider.UpdatePullRequest(gha, *existingPr.Number)
-		if err != nil {
-			return nil, errors.Wrapf(err, "updating pull request %s", existingPr.URL)
-		}
 		err = gitter.Push(dir, forkPushURL, true, false, fmt.Sprintf("%s:%s", "HEAD", prDetails.BranchName))
 		if err != nil {
 			return nil, errors.WithStack(err)
 		}
+		pr, err = provider.UpdatePullRequest(gha, *existingPr.Number)
```

Comment:
I've actually just spotted that we push on 311-315 and that this second push shouldn't even be there. Do you want to try just deleting it @abayer

Reply:
Well, yeesh. That’s hilarious. Updating.

Reply:
Done.